CONTRIBUTING.md

```diff
@@ -1,9 +1 @@
-- **All PRs should be made to the `devel` branch, not `master`.**
-
-- Come check-in in the [Telescope Slack channel](http://slack.telescopeapp.org/). 👋
-
-- Completely new features should be shipped as external packages with their own repos (see [3rd party packages](http://vulcan-docs.telescopeapp.org/plugins.html)). Don't hesitate to come by the [Slack channel](http://slack.telescopeapp.org/) to speak about it.
-
-- We don't have test at the moment, and Travis integration is broken. If you know how to fix it, you are welcome (see [#1253](https://github.com/TelescopeJS/Telescope/issues/1253)!
-
-- Be nice 😉
+TODO: Add contribution guidelines.
```

Comment:
I replaced this with a placeholder - the stuff from Telescope might lead to confusion.